packages/vuetify/src/components/VList/VList.tsx

```diff
@@ -92,6 +92,7 @@ export const makeVListProps = propsFactory({
     type: [Boolean, String] as PropType<'one' | 'two' | 'three' | false>,
     default: 'one',
   },
+  prependSpacer: [Number, String],
```

Comment:
What are your thoughts on calling this **slim-width**? Similar to how we do with v-navigation-drawer with **rail-width** and some others as well.

Reply:
Good idea, implemented.